sources/osgGA/OrbitManipulatorHammerController.js

```diff
@@ -123,19 +124,31 @@ OrbitManipulatorHammerController.prototype = {
             Notify.debug( 'zoom end, ' + dragCB( event ) );
         };
 
+        var smoothstep = function ( a, b, x ) {
```

Comment:
It would make sense to add this function in our glMatrix namespace and probably we will need to move later osgAnimation ease in osg core glMatrix, it's more math than animation actually